src/Symfony/Component/AssetMapper/ImportMap/Resolver/JsDelivrEsmResolver.php

```diff
@@ -326,4 +332,22 @@ private function makeImportsBare(string $content, array &$dependencies, array &$
 
         return preg_replace('{/\*# sourceMappingURL=[^ ]*+ \*/}', '', $content);
     }
+
+    /**
+     * Determine the URL pattern to be used by the HTTP Client.
+     */
+    private function resolveUrlPattern(string $packageName, string $path, ImportMapType $type = null): string
+    {
+        // The URL for the es-module-shims polyfill package uses the CSS pattern to
+        // prevent a syntax error in the browser console.
+        if (self::ES_MODULE_SHIMS === $packageName) {
+            return self::URL_PATTERN_DIST_CSS;
+        }
+
+        if (str_ends_with($path, '.css') || ($type && ImportMapType::CSS === $type)) {
```

Comment:
`$type && ` is useless as `null` won't be equal to `ImportMapType::CSS` anyway

Reply:
Hi, @broadcoder! If there's a future need for more `URL_PATTERN_DIST_CSS` exclusions based on package name, I think that approach makes total sense. I wonder if this is really just a single edge case? @weaverryan or others? If so, it might make sense for me to drop this PR and for you to submit yours. I *don't* want to hijack your effort. 

Sidenote: pretty sure I spent most of this PR trying to get that method name right, so props to thinking too alike. :)

Reply:
Thanks, @stof. I removed that null check.